src/js/utils/language.js

```diff
@@ -98,7 +98,8 @@ export function loadJsonTranslation(base, languageCode) {
     let translationLoad = translationPromises[languageCode];
     if (!translationLoad) {
         const url = `${base}translations/${normalizeLanguageCode(languageCode)}.json`;
-        translationPromises[languageCode] = translationLoad = new Promise((oncomplete, onerror) => {
+        translationPromises[languageCode] = translationLoad = new Promise((oncomplete, reject) => {
+            const onerror = (message, file, _url, error) => reject(error);
```

Comment:
When we do encounter an error, do we want to leave a rejected promise in `translationPromises[languageCode]`? That will automatically trigger the same error for the next player that sets up on a page instead of reattempting the request.

Reply:
True. how likely is it that the error will be a one off error vs a persistent error ? If it's persistent then we don't want to re try for every player.  Also would it be better to have consistency across players or is it better to have some translated and some not ? @aliciahurst any thoughts on the last question ?

Reply:
@robwalch never mind, addressed